src/lib.rs

```diff
@@ -26,6 +26,8 @@
 //! }
 //! ```
 
+use std::error::Error as StdError;
```

Comment:
I needed it somewhere so we can use the trait in the format call. I've mode it as close to the usage as possible. Let me know.